packages/wp-story-editor/src/components/checklist/accessibility/videoOptimizationCheckbox/index.js

```diff
@@ -25,18 +25,17 @@ import {
 import { TranslateWithMarkup, __ } from '@web-stories-wp/i18n';
 import { trackClick } from '@web-stories-wp/tracking';
 import styled from 'styled-components';
-/**
- * Internal dependencies
- */
-import { useConfig, useCurrentUser } from '../../../app';
-import { CARD_WIDTH } from '../../secondaryPopup';
-import { useIsChecklistMounted } from '../popupMountedContext';
+import {
+  useConfig,
+  useCurrentUser,
+  useIsChecklistMounted,
+} from '@web-stories-wp/story-editor';
 
 const Container = styled.div`
   display: flex;
   align-items: center;
   flex-direction: column;
-  width: ${CARD_WIDTH}px;
+  width: 272px;
```

Comment:
I wonder why this width was used in the first place.

Other checks don't need it because they use `<ChecklistCard>`.

Could we use `<ChecklistCard>` here as well instead of `<Container>`? Any style differences?

I'd want to avoid hardcoding 272px here.

Reply:
After and before respectively 👇 

![image](https://user-images.githubusercontent.com/6297436/142805296-628b7246-c894-4ebc-a8c0-01bef5335086.png)

Done https://github.com/google/web-stories-wp/pull/9710/commits/de15679a9ea6259eacb4228bce2d9ee6dbbeb649